mirage/views/preprint.ts

```diff
@@ -35,6 +36,8 @@ export function createPreprint(this: HandlerContext, schema: Schema) {
         subjects: [],
         tags: [] as string[] ,
         currentUserPermission: [Permission.Admin, Permission.Read, Permission.Write],
+        versionNumber: 1,
```

Comment:
And I would presume this needs to be 3 as well?

Reply:
This is in the `createPreprint` view, so the `versionNumber` property for a freshly created preprint should just be `1` in this case. I will have a separate view for `createPreprintVersion` that will increment the `versionNumber` property